From 264c9510015c65a6d8f4809c9da12901235705bc Mon Sep 17 00:00:00 2001
From: Martin Storsjo <martin@martin.st>
Date: Sat, 21 Sep 2019 19:09:43 +0000
Subject: [UPSTREAM] [LLDB] Fix compilation for MinGW, remove redundant class
 name on inline member

This fixes build errors like these:

NativeRegisterContextWindows.h:22:33: error: extra qualification on member 'NativeRegisterContextWindows'
  NativeRegisterContextWindows::NativeRegisterContextWindows(
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^

Differential Revision: https://reviews.llvm.org/D67856

Change-Id: I6257927879a92a0d48b740a0f956379e011f5acc
llvm-svn: 372482
(cherry picked from commit f4deacf995c73045a2172f35cbaf289fa97e7abc)
---
 .../Process/Windows/Common/NativeRegisterContextWindows.h       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lldb/source/Plugins/Process/Windows/Common/NativeRegisterContextWindows.h b/lldb/source/Plugins/Process/Windows/Common/NativeRegisterContextWindows.h
index ca3badd5cc0..841b8547f3e 100644
--- a/lldb/source/Plugins/Process/Windows/Common/NativeRegisterContextWindows.h
+++ b/lldb/source/Plugins/Process/Windows/Common/NativeRegisterContextWindows.h
@@ -19,7 +19,7 @@ class NativeThreadWindows;
 
 class NativeRegisterContextWindows : public NativeRegisterContextRegisterInfo {
 public:
-  NativeRegisterContextWindows::NativeRegisterContextWindows(
+  NativeRegisterContextWindows(
       NativeThreadProtocol &native_thread,
       RegisterInfoInterface *reg_info_interface_p);
 
-- 
2.25.0.341.g760bfbb309-goog

